Src/CSharpier.VSCode/src/CustomPathInstaller.ts

```diff
@@ -0,0 +1,38 @@
+import { Logger } from "./Logger";
+import * as path from "path";
+import * as fs from "fs";
+import { execSync } from "child_process";
+
+export class CustomPathInstaller {
+    logger: Logger;
+
+    constructor(logger: Logger) {
+        this.logger = logger;
+    }
+
+    public ensureVersionInstalled(version: string) {
+        if (!version) {
+            return;
+        }
+        const directoryForVersion = this.getDirectoryForVersion(version);
+        if (fs.existsSync(directoryForVersion)) {
+            this.logger.debug("File at " + directoryForVersion + " already exists");
+            return;
+        }
+
+        const command = "dotnet tool install csharpier --version " + version + " --tool-path";
```

Comment:
Uhm... shouldn't you have the path appended to this command after `--tool-path`?

Reply:
er, yeah. This probably worked with my testing because I forgot to delete everything from the custom path.